src/app/util/af-types.h

```diff
@@ -298,7 +310,10 @@ typedef struct
  */
 struct EmberAfClusterCommand
 {
-    chip::NodeId SourceNodeId() const { return source->GetSessionHandle()->AsSecureSession()->GetPeerNodeId(); }
+    chip::NodeId SourceNodeId() const
```

Comment:
Why the whitespace changes?  Makes review quite difficult, and restyled is not happy with them either...

Reply:
My local formatter misbahaved, reformatted.